### DISCUSSION TITLE
How to disable DAC output buffer on STM32F072CB?

### DISCUSSION THREAD

**Discussion by Bozog** [stm32duino/Arduino_Core_STM32#2305] [Q&A] [answered]
Hi, not sure if I'm being stupid, but I can't find any easy way to disable the DAC output buffer on my STM32F072CB
Ideas appreciated!

**Comment by fpistm** [comment, marked as answer]
Hi @Bozog 
It is configured here:
https://github.com/stm32duino/Arduino_Core_STM32/blob/641f3d8883e9a7220f65dded355dc32e218a990d/libraries/SrcWrapper/src/stm32/analog.cpp#L412

So not possible to change the value except if you change it manually.
If you want you can submit a PR to be able to redefine it, something like this:
```c
#if defined(DISABLE_DAC_OUTPUTBUFFER)
    dacChannelConf.DAC_OutputBuffer = DAC_OUTPUTBUFFER_DISABLE;
#else
    dacChannelConf.DAC_OutputBuffer = DAC_OUTPUTBUFFER_ENABLE;
#endif
```

Then you will be able to disable it by defining the `DISABLE_DAC_OUTPUTBUFFER` using `build_opt.h` or the `hal_conf_extra.h`

**Reply by Bozog** [reply]
Thanks for taking the time to reply, I did some testing in cube, turns out I don't need to disable it after all :)
There are certainly circumstances where disabling it would be beneficial although.
Regards
